src/utils.js

```diff
@@ -158,3 +158,16 @@ export function sorter ($sort) {
     return comparator;
   };
 }
+
+export function makeUrl (path, app = {}) {
+  const get = typeof app.get === 'function' ? app.get : () => {};
```

Comment:
Not sure I follow. This is primarily for building string types. It gets used in stuff [like this](https://github.com/feathersjs/feathers-authentication-oauth2/blob/master/src/index.js#L48).